UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -769,6 +835,22 @@
         });
       };
 
+      var sortAndFilterServicePointProperties = function(properties) {
+        var propertyOrdering = [
+            'palvelu',
+            'tarkenne',
+            'palvelun_nimi',
+            'palvelun_lisätieto',
+            'viranomaisdataa',
+            'suggest_box'];
+
+        return _.sortBy(properties, function(property) {
+          return _.indexOf(propertyOrdering, property.publicId);
+        }).filter(function(property){
+          return _.indexOf(propertyOrdering, property.publicId) >= 0;
+        });
+      };
```

Comment:
Use LodAsh here

Reply:
Used lodash and changed the way the ordering function works. Check code changes made to see if you agree